doc/man3/EVP_KDF.pod

```diff
@@ -124,11 +127,16 @@ simply ignored.
 Also, what happens when a needed parameter isn't passed down is
 defined by the implementation.
 
-EVP_KDF_gettable_params(), EVP_KDF_gettable_ctx_params() and
-EVP_KDF_settable_ctx_params() get a constant B<OSSL_PARAM> array that
-describes the retrievable and settable parameters, i.e. parameters that
-can be used with EVP_KDF_get_params(), EVP_KDF_CTX_get_params()
-and EVP_KDF_CTX_set_params(), respectively.
+EVP_KDF_gettable_params() returns parameters that can be used with
+EVP_KDF_get_params().  See L<OSSL_PARAM(3)> for the use of B<OSSL_PARAM>
+as parameter descriptor.
+
+EVP_KDF_gettable_ctx_params() and EVP_KDF_CTX_gettable_params()
+return parameters that can be used with EVP_KDF_CTX_get_params().
+See L<OSSL_PARAM(3)> for the use of B<OSSL_PARAM> as parameter descriptor.
+
+EVP_KDF_settable_ctx_params() and EVP_KDF_CTX_settable_params()
```

Comment:
This doesnt tell me what the diff between settable_ctx_params() and CTX_settable_params() is?